packages/transaction-controller/src/TransactionController.ts

```diff
@@ -258,6 +261,9 @@ export const CANCEL_RATE = 1.5;
  */
 export const SPEED_UP_RATE = 1.1;
 
+type Provider = any;
```

Comment:
I think these would end up being the proxies and not the "real" objects, right? Also, instead of redefining these types here, can we use ProviderProxy and BlockTrackerProxy from NetworkController? Probably have to export them.

Reply:
The type we use in the network controller at the moment is just `any`, so... in the short term this wouldn't do much good. But since we'll be improving it soon, maybe this is worth doing anyway.

Reply:
updated here: https://github.com/MetaMask/core/pull/1147/commits/431eedf05f6426a8bbe7f8b6e35981853f551b4b